store/src/main/java/com/dropbox/android/external/store4/impl/FetcherController.kt

```diff
@@ -83,8 +83,6 @@ internal class FetcherController<Key : Any, Input : Any, Output : Any>(
                             origin = ResponseOrigin.Fetcher
                         )
                     }
-                }.onEmpty {
-                    emit(StoreResponse.NoNewData(ResponseOrigin.Fetcher))
```

Comment:
this still needs to be addressed - if the fetcher is empty we will never revert from loading state to non loading state